merlin/analysis/optimize.py

```diff
@@ -32,6 +32,12 @@ def __init__(self, dataSet, parameters=None, analysisName=None):
             self.parameters['optimize_chromatic_correction'] = False
         if 'crop_width' not in self.parameters:
             self.parameters['crop_width'] = 0
+        if 'fov_index' in self.parameters:
+            # Set iterations number to length of fov_index if not
+            # None. Should we warn the user?
```

Comment:
I think we can just mention in the documentation that fov_index takes precedent over fov_per_iteration. We could also try logging a message to the logger from dataSet.get_logger(self). This logging could still have bugs since I don't use it extensively.